site/lib/color.js

```diff
@@ -9,7 +9,7 @@ export const noCasesColor = '#ffffff';
 export const noPopulationDataColor = 'rgba(0,0,0,0)';
 
 export const outlineColorHighlight = 'rgb(0,0,0)';
-export const outlineColor = 'rgba(0, 0, 0, 0.3)';
+export const outlineColor = 'rgba(0, 0, 0, 0)';
```

Comment:
This should be in a different PR.

Reply:
oops - sorry about that! snuck in from #419